.storybook/manager.js

```diff
@@ -0,0 +1,6 @@
+import { addons } from '@storybook/addons';
+import USWDS from './USWDS';
+
+addons.setConfig({
+  theme: USWDS,
+});
```

Comment:
This needs a newline.

Reminds me I need to get prettier working in CI or something since our pre-commit checks no longer run

Reply:
fixed 👍